include/revng/Pipeline/ContainerFactory.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+//
```

Comment:
This file is probably no longer worth a dedicated header. Consider inlining it somewhere appropriate.